packages/client/mutations/EndNewMeetingMutation.ts

```diff
@@ -24,7 +26,19 @@ graphql`
       id
       endedAt
       teamId
+      ... on ActionMeeting {
+        agendaItemCount
+        commentCount
+        taskCount
+      }
+      ... on RetrospectiveMeeting {
+        commentCount
+        reflectionCount
+        taskCount
+        topicCount
+      }
```

Comment:
Where do you need these fields?

Reply:
They're being used in `TimelineEventCompletedActionMeeting.tsx` & `...RetroMeeting.tsx`. In the `endNewMeetingTeamUpdater` function, I'm setting the linkedRecord of `meeting` on to `timelineEvent` so that we have access to these fields in the frontend.